src/ui/field_formats/__tests__/field_formats_mixin.js

```diff
@@ -27,26 +27,26 @@ import { createServer } from '../../../test_utils/kbn_server';
 describe('server.registerFieldFormat(createFormat)', () => {
   const sandbox = sinon.createSandbox();
 
-  let server;
+  let kbnServer;
   beforeEach(async () => {
-    const kbnServer = createServer();
+    kbnServer = createServer();
```

Comment:
Is this the only place we were using `createServer()` that wasn't calling `kbnServer.close()` which I assume is only really necessary because of the change to `autoListen: true` by default.